website/docs/r/pi_virtual_serial_number.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_virtual_serial_number"
+description: |-
+  Manages an EXISTING virtual serial number in IBM Power
+---
+
+# ibm_pi_virtual_serial_number
+Get, update or delete an existing virtual serial number.
+
+## Example usage
+The following example enables you to create a virtual serial number:
+
+```terraform
+resource "ibm_pi_virtual_serial_number" "testacc_virtual_serial_number" {
+  pi_serial            = "<existing virtual serial number>"
+  pi_cloud_instance_id = "<value of the cloud_instance_id>"
+  pi_description       = "<desired description for virtual serial number>"
+}
+```
+
+### Notes
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+- If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  - `region` - `lon`
+  - `zone` - `lon04`
+  
+  Example usage:
+
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+**Note**
+- This resource is used to create a virtual serial number by assigning to a power instance using 'auto-assign'. Otherwise, it can only be used to manage an existing virtual serial number.
+
+## Timeouts
+
+ibm_pi_virtual_serial_number provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:
+
+- **create** - (Default 45 minutes) Used for creating an existing virtual serial number.
+- **update** - (Default 45 minutes) Used for updating a virtual serial number.
+- **delete** - (Default 45 minutes) Used for deleting a reserved virtual serial number.
+
```

Comment:
The existing language can be updated now.

Reply:
I have it because you can specify an existing virtual serial number. If not assigning to an instance, you cannot create one. That is why I have it in there.

Reply:
Line 46:  Is what I was referring to, the Note is fine.
```suggestion

**Note**
- This resource is used to create a virtual serial number by assigning to a power instance using 'auto-assign'. Otherwise, it can only be used to manage an existing virtual serial number.

## Timeouts

ibm_pi_virtual_serial_number provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:

- **create** - (Default 45 minutes) Used for creating a virtual serial number.
- **update** - (Default 45 minutes) Used for updating a virtual serial number.
- **delete** - (Default 45 minutes) Used for deleting a reserved virtual serial number.

```